teamC/sungsik/models/userDao.js

```diff
@@ -0,0 +1,53 @@
+const { DataSource } = require('typeorm');
+
+const database = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+database.initialize()
+
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+    .catch((err) => {    
+        console.error("Error occurred during Data Source initilization", err);
+        database.destroy()
+    })
```

Comment:
지금은 postDao, userDao 파일 마다 데이터베이스 객체를 생성하고 있습니다.
`/models/data-source.js` 파일에 아래 코드를 옮겨주시고, dao 파일에서는 database를 import해서 사용하는 방식으로 수정해주세요~

이렇게 하는 이유는 매번 데이터베이스 객체를 생성하지 않고, 데이터베이스와의 컨넥션도 같이 사용할 수 있습니다.

```suggestion
const database = new DataSource({
    type: process.env.TYPEORM_CONNECTION,
    host: process.env.TYPEORM_HOST,
    port: process.env.TYPEORM_PORT,
    username: process.env.TYPEORM_USERNAME,
    password: process.env.TYPEORM_PASSWORD,
    database: process.env.TYPEORM_DATABASE
})

database.initialize()

    .then(() => {
        console.log("Data Source has been initialized!")
    })
    .catch((err) => {    
        console.error("Error occurred during Data Source initilization", err);
        database.destroy()
    })
```